src/components/address-input/_macro.spec.js

```diff
@@ -4,69 +4,22 @@ import * as cheerio from 'cheerio';
 
 import axe from '../../tests/helpers/axe';
 import { renderComponent, templateFaker } from '../../tests/helpers/rendering';
+import { EXAMPLE_AUTOSUGGEST_ADDRESS_MINIMAL, EXAMPLE_MANUAL_INPUT_FIELDS } from './_test-examples';
 
-const EXAMPLE_AUTOSUGGEST_ADDRESS_MINIMAL = {
-    id: 'address-input-example-id',
-    legend: 'What is the address?',
-    label: {
-        text: 'Enter address or postcode and select from results',
-        id: 'address-input-example-label-id',
-    },
-    isEditable: false,
-    instructions: 'Use up and down keys to navigate suggestions.',
-    ariaYouHaveSelected: 'You have selected',
-    ariaMinChars: 'Enter 3 or more characters for suggestions.',
-    ariaOneResult: 'There is one suggestion available.',
-    ariaNResults: 'There are {n} suggestions available.',
-    ariaLimitedResults: 'Results have been limited to 10 suggestions. Type more characters to improve your search',
-    ariaGroupedResults: 'There are {n} for {x}',
-    groupCount: '{n} addresses',
-    moreResults: 'Enter more of the address to improve results',
-    noResults: 'No results found. Try entering a different part of the address',
-    tooManyResults: '{n} results found. Enter more of the address to improve results',
-    typeMore: 'Enter more of the address to get results',
-    resultsTitle: 'Select an address',
-    resultsTitleId: 'address-suggestions',
-};
-
-const EXAMPLE_MANUAL_INPUT_FIELDS = {
-    organisation: {
-        label: 'Organisation name',
-        value: 'Example Organisation',
-        error: { text: 'Server error: organisation name' },
-    },
-    line1: {
-        label: 'Address line 1',
-        value: 'Flat 12345',
-        error: { text: 'Server error: address line 1' },
-    },
-    line2: {
-        label: 'Address line 2',
-        value: '12345 The Road',
-        error: { text: 'Server error: address line 2' },
-    },
-    town: {
-        label: 'Town or city',
-        value: 'The Town',
-        error: { text: 'Server error: town or city' },
-    },
-    postcode: {
-        label: 'Postcode',
-        value: 'PO57 6ODE',
-        error: { text: 'Server error: postcode' },
-    },
-};
-
-describe('macro: address-input', () => {
-    it('passes jest-axe checks', async () => {
-        const $ = cheerio.load(renderComponent('address-input', EXAMPLE_AUTOSUGGEST_ADDRESS_MINIMAL));
-
-        const results = await axe($.html());
-        expect(results).toHaveNoViolations();
+describe('FOR: address-input', () => {
```

Comment:
```suggestion
describe('FOR: Macro: Address-input', () => {
```
Same comment as other PR